base/set.jl

```diff
@@ -1,5 +1,33 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+	Set{T} <: AbstractSet{T}
+
+The default implementation of hash sets.
```

Comment:
"hash set" seems a little technical for the first line. Can it just explain somehow, that membership is the only concept? Explain `isequal` and `hash` later.

Reply:
To me, this sentence doesn't really add anything that the text below already explains.